sdk/translation/azure-ai-translation-document/src/main/java/com/azure/ai/translation/document/models/TranslationSource.java

```diff
@@ -0,0 +1,271 @@
+        return this;
+    }
+
+    /*
+     * Language code
+     * If none is specified, we will perform auto detect on the document
+     */
+    @Generated
+    private String language;
+
+    /*
+     * Storage Source
+     */
+    @Generated
+    private TranslationStorageSource storageSource;
+
+    /**
+     * Creates an instance of TranslationSource class.
+     *
+     * @param sourceUrl the sourceUrl value to set.
+     */
+    @Generated
+    public TranslationSource(String sourceUrl) {
+        this.sourceUrl = sourceUrl;
+    }
+
+    /**
+     * Get the sourceUrl property: Location of the folder / container or single file with your documents.
+     *
+     * @return the sourceUrl value.
+     */
+    @Generated
+    public String getSourceUrl() {
+        return this.sourceUrl;
+    }
+
+    /**
+     * Get the filter property: Document filter.
+     *
+     * @return the filter value.
+     */
+    @Generated
+    public DocumentFilter getFilter() {
+        return this.filter;
+    }
+
+    /**
+     * Set the filter property: Document filter.
+     *
+     * @param filter the filter value to set.
+     * @return the TranslationSource object itself.
+     */
+    @Generated
+    public TranslationSource setFilter(DocumentFilter filter) {
+        this.filter = filter;
+        return this;
+    }
+
+    /**
+     * Get the language property: Language code
+     * If none is specified, we will perform auto detect on the document.
+     *
+     * @return the language value.
+     */
+    @Generated
+    public String getLanguage() {
+        return this.language;
+    }
+
+    /**
+     * Set the language property: Language code
+     * If none is specified, we will perform auto detect on the document.
+     *
+     * @param language the language value to set.
+     * @return the TranslationSource object itself.
+     */
+    @Generated
+    public TranslationSource setLanguage(String language) {
+        this.language = language;
+        return this;
+    }
+
+    /**
+     * Get the storageSource property: Storage Source.
+     *
+     * @return the storageSource value.
+     */
+    @Generated
+    public TranslationStorageSource getStorageSource() {
+        return this.storageSource;
+    }
+
+    /**
+     * Set the storageSource property: Storage Source.
+     *
+     * @param storageSource the storageSource value to set.
+     * @return the TranslationSource object itself.
+     */
+    @Generated
+    public TranslationSource setStorageSource(TranslationStorageSource storageSource) {
+        this.storageSource = storageSource;
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
+        jsonWriter.writeStringField("sourceUrl", this.sourceUrl);
+        jsonWriter.writeStringField("prefix", this.prefix);
+        jsonWriter.writeStringField("suffix", this.suffix);
```

Comment:
This will not work as this is changing the shape of the json that is sent to the service. We should not change the json payload structure.

Reply:
Reverted this change